compiler/rustc_middle/src/ty/context.rs

```diff
@@ -2059,8 +2056,6 @@ slice_interners!(
     predicates: _intern_predicates(Predicate<'tcx>),
     projs: _intern_projs(ProjectionKind),
     place_elems: _intern_place_elems(PlaceElem<'tcx>),
-    chalk_environment_clause_list:
```

Comment:
The existing method for predicates are used instead, see also the other edits in this file.